docs/guide/index.md

```diff
@@ -34,10 +34,8 @@ const store = createStore({
   }
 })
 
-const app = createApp({ /* your root component */ })
-
 // Install the store instance as a plugin
-app.use(store)
+createApp({ /* your root component */ }).use(store)
```

Comment:
creating a Vue instance is not a part of install a plugin, there is no need to bind the creation of Vue instance with the installation of store plugin.